.git-blame-ignore-revs

```diff
@@ -0,0 +1,4 @@
+# git config blame.ignoreRevsFile .git-blame-ignore-revs
```

Comment:
It may be worth adding a more detailed comment explaining that this command NEEDS to be run locally in order for future `git blame` commands to ignore commits in this file as intended, and that it doesn't just work by default.

Reply:
good call. updated